SCClassLibrary/JITLib/ProxySpace/wrapForNodeProxy.sc

```diff
@@ -296,11 +296,14 @@
 				};
 
 				{ | out |
-					var e = EnvGate.new * Control.names(["wet"++(index ? 0)]).kr(1.0);
+
```

Comment:
I think it would be better not to have an empty line here...?

Reply:
removed